lib/matplotlib/table.py

```diff
@@ -29,9 +29,38 @@
 
 class Cell(Rectangle):
     """
-    A cell is a  `.Rectangle` with some associated text.
+    A cell is a  `.Rectangle` with some associated `.Text`.
+
+    .. note:
+        As a user, you'll most likely not creates cells yourself. Instead, you
+        should use either the `~matplotlib.table.table` factory function or
+        `.Table.add_cell`.
+
+    Parameters
+    ----------
+    xy : 2-tuple
+        The position of the bottom left corner of the cell.
+    width : float
+        The cell width.
+    height : float
+        The cell height.
+    edgecolor : color spec
+        The color of the cell border.
+    facecolor : color spec
+        The cell facecolor.
+    fill : bool
+        Whether the cell background is filled.
+    text : str
+        The cell text.
+    loc : {'left', 'center', 'right'}, default: 'right'
+        The alignment of the text within the cell.
+    fontproperties : dict
+        A dict defining the font properties of the text. Supported keys and
+        values are the keyword arguments accepted by `.FontProperties`.
     """
-    PAD = 0.1  # padding between text and rectangle
+
+    PAD = 0.1
+    """Padding between text and rectangle."""
```

Comment:
Kind of prefer leaving this as inline comment (fwiw).
(As a side note, compare with the padding of Annotations, which is also hardcoded
```
                    pad = renderer.points_to_pixels(4)
```
)

Reply:
Since this is a public attribute, it [shows up in the docs](https://matplotlib.org/devdocs/api/table_api.html?highlight=table#matplotlib.table.Cell.PAD)

I very much prefer that every element show the docs has a description. This is achived [using the docstring](https://15585-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/table_api.html?highlight=table#matplotlib.table.Cell.PAD)

I would be fine not documenting `PAD`, but only if we also remove it from the docs (either by excluding explicitly (not sure if that's feasible with autodoc in this context), or preferably by renaming to `_PAD`.

Reply:
Oh, I didn't realize sphinx would pick it up.  I guess that works, then...